sdk/src/TrancheConfig.ts

```diff
@@ -0,0 +1,91 @@
+import { PublicKey } from "@solana/web3.js";
+import { TrancheDataType, ReserveFairValueType, SlotTrackingType, LastUpdateType, TrancheFairValueType } from './types/Tranche'
+
+export class TrancheConfig {
```

Comment:
can we have a single class per file? for example:
TrancheConfig.ts -> `export class TrancheConfig`
TrancheData.ts -> `export class TrancheData`

Reply:
Sounds good, doing it.